package.json

```diff
@@ -63,25 +63,14 @@
   "author": "Datt Developers <developers@datt.co>",
   "license": "MIT",
   "dependencies": {
-    "fullnode": "^0.10.0"
+    "async": "^1.3.0",
+    "html-metadata": "^1.2.1",
+    "maki": "git://github.com/martindale/maki",
+    "maki-passport-local": "0.0.3"
```

Comment:
I plan on replacing this with `maki-identity-mnemonic`, but I haven't yet finished this module.  Working on that later this week.